rmgpy/chemkinTest.py

```diff
@@ -118,25 +118,26 @@ def test_readThermoEntry_NoTRange(self):
         self.assertEqual(formula, {'H': 6, 'C': 2})
         self.assertTrue(isinstance(thermo, NASA))
 
-    def testReadAndWriteTemplateReactionFamilyForMinimalExample(self):
+    def testReadAndWriteAndReadTemplateReactionFamilyForMinimalExample(self):
         """
-        This example is mainly to test if family info can be correctly
+        This example tests if family and templates info can be correctly
         parsed from comments like '!Template reaction: R_Recombination'.
         """
         folder = os.path.join(os.path.dirname(rmgpy.__file__), 'test_data/chemkin/chemkin_py')
 
         chemkinPath = os.path.join(folder, 'minimal', 'chem.inp')
         dictionaryPath = os.path.join(folder, 'minimal', 'species_dictionary.txt')
 
-        # loadChemkinFile
+        # read original chemkin file
         species, reactions = loadChemkinFile(chemkinPath, dictionaryPath)
 
+        # ensure correct reading
         reaction1 = reactions[0]
         self.assertEqual(reaction1.family, "R_Recombination")
-
+        self.assertEqual(frozenset('C_methyl;C_methyl'.split(';')), frozenset(reaction1.template))
```

Comment:
Why can't we compare the templates directly?

Reply:
The order of the list is not deterministic, so comparing lists with different orders could lead to false Negative.